lib/sets-lookup.ts

```diff
@@ -0,0 +1,35 @@
+export const INTS = (): Record<string | number, boolean> => ({
+  '48-57': true,
+});
+
+export const WORDS = (): Record<string | number, boolean> => ({
+  95: true,
+  '97-122': true,
+  '65-90': true,
+  '48-57': true,
+});
+
+export const WHITESPACE = (): Record<string | number, boolean> => ({
+  9: true,
+  10: true,
+  11: true,
+  12: true,
+  13: true,
+  32: true,
+  160: true,
+  5760: true,
+  '8192-8202': true,
+  8232: true,
+  8233: true,
+  8239: true,
+  8287: true,
+  12288: true,
+  65279: true,
+});
+
+export const NOTANYCHAR = (): Record<string | number, boolean> => ({
+  10: true,
+  13: true,
+  8232: true,
+  8233: true,
+});
```

Comment:
since these are all generated here on each `isSameSet` call, could be worth it to generate these from `lib/sets.ts` inistead, to avoid having to keep both files synced up if there are changes to the js spec, or there are new sets.

what do you think? that way, we could also get their length from there

Reply:
Makes sense to me. Im assuming the lengths would be exported as constants?

Additionally do you want this generated at runtime, or should I use something like (ts-morph)[https://ts-morph.com/] to generate the file as part of the build script.

Reply:
no need for ts-morph, they can be generated at runtime. the current method does that, so it would be the same speed.

Reply:
> Makes sense to me. Im assuming the lengths would be exported as constants?

ah yes, they don't currently contain length until after they're called 😅

Reply:
> no need for ts-morph, they can be generated at runtime. the current method does that, so it would be the same speed.
Yikes, only just saw this after playing around doing with ts-morph sweat_smile. Will revert latest commits and push other solution tommorow.